src/group_theory/nilpotent.lean

```diff
@@ -131,6 +131,16 @@ instance (n : ℕ) : normal (upper_central_series G n) := (upper_central_series_
 
 @[simp] lemma upper_central_series_zero : upper_central_series G 0 = ⊥ := rfl
 
+@[simp] lemma upper_central_series_one : upper_central_series G 1 = center G :=
+begin
+  ext,
+  simp [upper_central_series, upper_central_series_aux, upper_central_series_step,
+    center, set.center],
+  apply forall_congr,
+  intro y,
+  rw [mul_inv_eq_one, mul_inv_eq_iff_eq_mul, eq_comm ],
+end
```

Comment:
Another non-terminal simp. It would be fine if it's just `rw`?

Reply:
A non-terminal `rw` is certainly fine but another option is to use `simp only` instead of `simp`. Furthermore, you can use `squeeze_simp` to turn a `simp` into a `simp only` (there are some bugs but it usually works). I used `squeeze_simp` to generate the suggestion here.